examples/docker/modin-hdk/census-hdk.py

```diff
@@ -14,7 +14,7 @@
 import sys
 import time
 import modin.pandas as pd
-from modin.experimental.core.execution.native.implementations.omnisci_on_native.omnisci_worker import (
```

Comment:
`DbWorker` class alias is declared in `db_worker.py`. Looks like a wrong import was used here after adding HDK support. A good moment to fix that )